pages/categories.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import Button from "@material-ui/core/Button";
+
+function Categories() {
```

Comment:
This function lacks a return type, if it should be proper Typescript. See react docs for more information.